packages/wrangler/src/init.ts

```diff
@@ -58,6 +59,12 @@ export function initOptions(yargs: CommonYargsArgv) {
 				"The name of the Worker you wish to download from the Cloudflare dashboard for local development.",
 			type: "string",
 			requiresArg: true,
+		})
+		.option("do-not-delegate", {
```

Comment:
could we name this flag without a negation in the name? negation often ends up being used with double negation, which confuses many people. I find that positive flag names are usually more clear and easier to use.